database-plugins/docs/Database-batchsource.md

```diff
@@ -3,46 +3,41 @@
 
 Description
 -----------
-Reads from a database using a configurable SQL query.
-Outputs one record for each row returned by the query.
+Database is a Guavus Enterprise Accelerator that is used to read from a database using a configurable SQL query. It outputs one record for each row returned by the query.
```

Comment:
this is not "Guavus Enterprise Accelerator" . it is an "open source accelerator" .